jupyterlab_bigquery/src/components/details_panel/table_preview.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react';
+import { stylesheet } from 'typestyle';
+
+import {
+  Table,
+  TableBody,
+  TableHead,
+  TableCell,
+  TableRow,
+  TablePagination,
+} from '@material-ui/core';
+
+import LoadingPanel from '../loading_panel';
+import {
+  TableDetailsService,
+  TablePreview,
+} from './service/list_table_details';
+
+const localStyles = stylesheet({
```

Comment:
Since these are very similar to the table styles for the query editor results, I will pull out a shared table style in a future PR